wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Data.kt

```diff
@@ -165,8 +177,40 @@ data class SignerWeight(
   val recoveryServer: Int,
 )
 
+enum class StellarOperationTypes(val type: String) {
```

Comment:
nit: this and others should be renamed to singular (StellarOperationType), it's a standard naming convention

Reply:
Yes, Jamie mentioned that to me. :blush: Will update!